chart/flux/templates/helm-operator-crd.yaml

```diff
@@ -44,13 +44,45 @@ spec:
               type: boolean
             valueFileSecrets:
               type: array
-              properties:
-                items:
-                  type: object
-                  required: ['name']
-                  properties:
-                    name:
-                      type: string
+              items:
+                type: object
+                required: ['name']
+                properties:
+                  name:
+                    type: string
+            valuesFrom:
+              type: array
+              items:
+                type: object
+                properties:
```

Comment:
Should there be a `oneOf` somewhere around here, to restrict each item to be _either_ a `secretKeyRef` or a `configMapKeyRef` or an `externalSourceRef`?

Reply:
I ended up having quite a fight yesterday with `oneOf` and could not get it to work properly in the end. Will make a second attempt...